src/app/dialogs/request-dialog/request-dialog.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="dialog-content-wrapper">
+  <div class="dialog-top">
+    <mat-icon class="icon green-text" mat-dialog-close>close</mat-icon>
+  </div>
+  <h2>Please, leave a message and your achievement will be assigned soon</h2>
+  <mat-dialog-content>
+    <form [formGroup]="requestAchievementForm">
+      <mat-form-field>
+        <mat-label>Achievement</mat-label>
+        <mat-select formControlName="achievement">
+          <mat-option value="1">value 1</mat-option>
+          <mat-option value="2">value 2</mat-option>
+          <mat-option value="3">value 3</mat-option>
+        </mat-select>
+      </mat-form-field>
+
+      <mat-form-field>
+        <mat-label>Your message</mat-label>
+        <textarea matInput formControlName="message"></textarea>
+      </mat-form-field>
+      <div class="center-content">
+        <button class="btn block-button" type="submit" [mat-dialog-close]="requestAchievementForm.value"
+          [disabled]="!requestAchievementForm.valid">Send</button>
+      </div>
+    </form>
+  </mat-dialog-content>
+
+</div>
```

Comment:
```suggestion
<div class="dialog-content-wrapper">
  <div class="dialog-top">
    <mat-icon class="icon green-text" mat-dialog-close>close</mat-icon>
  </div>
  
  <h2>Please, leave a message and your achievement will be assigned soon</h2>
  
  <mat-dialog-content>
    <form [formGroup]="requestAchievementForm">
      <mat-form-field>
        <mat-label>Achievement</mat-label>
        
        <mat-select formControlName="achievement">
          <mat-option value="1">value 1</mat-option>
          
          <mat-option value="2">value 2</mat-option>
          
          <mat-option value="3">value 3</mat-option>
        </mat-select>
      </mat-form-field>

      <mat-form-field>
        <mat-label>Your message</mat-label>
        
        <textarea matInput formControlName="message"></textarea>
      </mat-form-field>
      
      <div class="center-content">
        <button class="btn block-button" type="submit" [mat-dialog-close]="requestAchievementForm.value"
          [disabled]="!requestAchievementForm.valid">Send</button>
      </div>
    </form>
  </mat-dialog-content>
</div>
```

this formatting example. please check it